File: packages/core/src/models/ProviderConfig.ts
/**
 * Конфигурация провайдера AI-сервиса
 */
export type ProviderType = "openai" | "anthropic" | "gemini" | "zai";

export interface ProviderConfig {
  id: string;
  type: ProviderType;
  apiKey?: string;
  customEndpoint?: string;
  displayName?: string;
  /** Ручной лимит (для z.ai, Gemini без API) — число в unit */
  manualLimit?: number;
  /** Ручное значение used (для z.ai) */
  manualUsed?: number;
}

File: apps/android/app/src/main/java/com/ailimits/dashboard/data/ProviderConfig.kt
package com.ailimits.dashboard.data

data class ProviderConfig(
    val id: String,
    val type: ProviderType,
    val apiKey: String? = null,
    val customEndpoint: String? = null,
    val displayName: String? = null,
    val manualLimit: Double? = null,
    val manualUsed: Double? = null,
)

enum class ProviderType { OPENAI, ANTHROPIC, GEMINI, ZAI }
